app/broadcast_message_history/rest.py

```diff
@@ -0,0 +1,37 @@
+from flask import Blueprint, jsonify
+
+from app.dao.broadcast_message_history_dao import (
+    dao_get_broadcast_message_by_id_service_id_and_version_number,
+    dao_get_broadcast_message_versions,
+)
+from app.errors import register_errors
+from app.schemas import broadcast_message_history_schema
+
+broadcast_message_history_blueprint = Blueprint(
+    "broadcast_message_history", __name__, url_prefix="/service/<uuid:service_id>/broadcast-message-history"
+)
+register_errors(broadcast_message_history_blueprint)
+
+
+@broadcast_message_history_blueprint.route("/<uuid:broadcast_message_id>/version/<int:version>")
+def get_broadcast_message_version(service_id, broadcast_message_id, version):
+    data = broadcast_message_history_schema.dump(
+        dao_get_broadcast_message_by_id_service_id_and_version_number(
+            broadcast_message_id=broadcast_message_id, service_id=service_id, version=version
+        )
+    )
+    return jsonify(data)
+
+
+@broadcast_message_history_blueprint.route("/<uuid:broadcast_message_id>/versions")
+def get_broadcast_message_versions(service_id, broadcast_message_id):
+    broadcast_messages = [
+        {
+            **message.serialize(),
+            "created_by": creatd_by or None,
+        }
+        for message, creatd_by in dao_get_broadcast_message_versions(
+            service_id=service_id, broadcast_message_id=broadcast_message_id
+        )
+    ]
```

Comment:
I know this is quite compact, but it's awkward to read with the trailing loop.  You end up having to read the code backwards to understand it properly.  I'd be tempted to just build up the array with a map or use a standard for loop and appending.

Any particular reason it's creatd rather than created?